WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -2812,6 +2812,12 @@ k_poland = {
 	color={ 150 23 23 }
 	color2={ 200 0 0 }
 }
+k_lithuania = {
+	color = { 75 125 6 }
+	color2 = { 255 255 255 }
+	
+	capital = 420 # Aukshayts/Aukszota
+}
```

Comment:
Why did you add this kingdom?

Reply:
It was missing and similar kingdoms (Pomerania, Poland etc.) are also here

Reply:
Missing from what? Is there a Lithuania in vanilla?

Reply:
It should also be possible to form this kingdom in later game

Reply:
I think I took this from vanilla but I'm not sure, I will check

Reply:
Yes, there is

Reply:
Okay.